### ISSUE TITLE
dts 1.0.7 'file not found'

### ISSUE THREAD

**Issue by codyjs** [inversify/InversifyJS#269]
## Expected Behavior

Transpiling code which depends on inversify-express-utils.d.ts should work

## Current Behavior

Transpilation fails due to error in .d.ts

## Possible Solution

Remove unnecessary `/// <reference />` tag here:
https://github.com/inversify/inversify-dts/blob/master/inversify-express-utils/inversify-express-utils.d.ts#L6

## Steps to Reproduce (for bugs)

After attempting to update inversify-express-example with most recent inversify-dts and inversify-express-utils, running ts-node BasicExample/bootstrap produced the error

## Your Environment

- Version used: inversify-express-utils: 1.0.0-beta.2; inversify-dts 1.0.7
# Stack trace

```
node_modules\inversify-dts\inversify-express-utils\inversify-express-utils.d.ts (6,1): File 'C:/Users/Cody/projects/inversify-express-example/node_modules/inversify-dts/typings/index.d.ts' not found. (6053)
```

**Comment by lholznagel** [comment]
Interesting, just updated the example. The only problem was that I forgot to install `inversify-dts`. After installing it all worked

**Comment by lholznagel** [comment]
Oh, now I see the problem, forgot to take a look if the server starts. When I try to start it I get the same message.

**Comment by remojansen** [comment]
Hi guys, thanks for all the work! I merged https://github.com/inversify/inversify-dts/pull/5 and released `inversify-dts@1.0.8` please let me know if this issue can be closed.

**Comment by lholznagel** [comment]
In the express example everything works :)
